recipes/gromacs/4.6.5/meta.yaml

```diff
@@ -0,0 +1,27 @@
+about:
+  home: http://www.gromacs.org/
+  license: GNU Lesser General Public License (LGPL)
+  summary: GROMACS is a versatile package to perform molecular dynamics.
+
+package:
+  name: gromacs
+  version: 4.6.5
+
+build:
+  number: 0
+  skip: True #  [osx]
```

Comment:
Let travis test it on OS-X. If you see problems you can add this skip step later on :)